LvivCompany.Bookstore/LvivCompany.Bookstore.Web/appsettings.json

```diff
@@ -1,8 +1,8 @@
 ﻿{
   "ConnectionStrings": {
-    "DefaultConnectionToIdentityDb": "Server=(localdb)\\mssqllocaldb;Database=Identitydb;Trusted_Connection=True;MultipleActiveResultSets=true",  
+    "DefaultConnectionToIdentityDb": "Server=(localdb)\\mssqllocaldb;Database=Identitydb;Trusted_Connection=True;MultipleActiveResultSets=true",
     "DefaultConnectionToDb": "Server=(localdb)\\mssqllocaldb;Database=Identitydb;Trusted_Connection=True;MultipleActiveResultSets=true",
-    "DefaultConnection": "Server=(localdb)\\mssqllocaldb;Database=Identitydb;Trusted_Connection=True;MultipleActiveResultSets=true"
+    "DefaultConnection": "Server=DESKTOP-ANNA\\SQLEXPRESS;Database=bookstoredb;Trusted_Connection=True;MultipleActiveResultSets=true"
```

Comment:
Don't commit user specific connection string. It should work for all team members